service/src/test/java/edu/tamu/catalog/controller/PatronControllerTest.java

```diff
@@ -444,11 +444,11 @@ private void performPatronGetWithMockMVC(String url, String endpoint, PathParame
             Arguments.of(block, between(0, 1), once(), once(), respondJsonOk(blUserBadUUIDErrorPayload),
                 withStatus(BAD_REQUEST), "Bad%20UUID", status().isBadRequest()),
             Arguments.of(block, between(0, 1), once(), never(), respondJsonOk(blockEmptyCatalogPayload),
-                withNoContent(), USER_ID, status().isNotFound()),
+                withNoContent(), USER_ID, status().isInternalServerError()),
             Arguments.of(block, between(0, 1), once(), between(0, 1), respondJsonOk(blUserDuplicateErrorPayload),
                 respondJsonOk(automatedBlocksResponsePayload), USER_ID, status().isInternalServerError()),
             Arguments.of(block, between(0, 1), once(), between(0, 1), respondJsonOk(blUserEmptyErrorPayload),
-                respondJsonOk(automatedBlocksResponsePayload), USER_ID, status().isNotFound()));
+                respondJsonOk(automatedBlocksResponsePayload), USER_ID, status().isInternalServerError()));
```

Comment:
If we are transforming the 404 from FOLIO to a 500, then this test is fine and is actually redundant and should be removed (because line 450 is already doing this).

For reference, arguments for the test stream are the function arguments here:
https://github.com/TAMULib/CatalogService/blob/4e49dae8c5ce12c978b10b6f64df158c40e65ee1/service/src/test/java/edu/tamu/catalog/controller/PatronControllerTest.java#L337

Reply:
It is somewhat redundant but handles the unlikely FOLIO response that the user is found but the user does not have an id. I was curious in why you provided `blUserEmptyErrorPayload`. But I went with it anyhow as it is just another safety check.